packages/models/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "tsconfig/dtos.json",
```

Comment:
it should not work, you have not added "tsconfig": "*" to the devDependencies

Reply:
Added